### PR TITLE
editorial for directory traversal

### PR DESCRIPTION
**Pull request by ryanchou-dev**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

### CODE REVIEW

File: solutions/gold/usaco-814.mdx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+<LanguageSection>
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+struct Data {
+	bool isFile;  // is it a leaf
```
Comment (SansPapyrus683) [inline solutions/gold/usaco-814.mdx:76]: snake_case

File: solutions/gold/usaco-814.mdx [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+		for (int j = 0; j < m; j++) {
+			int x;
+			cin >> x;
+			--x;
+			bessie[i].children.push_back(x);
+		}
+	}
+
+	// compute insides & then outsides
```
Comment (SansPapyrus683) [inline solutions/gold/usaco-814.mdx:110]: maybe put "compute insides" then "and then compute outsides" as comments
idk might flow better (optional)

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: solutions/gold/usaco-814.mdx (hunk 2 above)
Comment (ryanchou-dev) [reply solutions/gold/usaco-814.mdx:110]: gotcha

File: solutions/gold/usaco-814.mdx (hunk 1 above)
Comment (ryanchou-dev) [reply solutions/gold/usaco-814.mdx:76]: i have never done that before i don't know what's gotten into me.

**Review by SansPapyrus683 — APPROVED** [review]
My name is Walter Hartwell White. I live at 308 Negra Arroyo Lane, Albuquerque, New Mexico, 87104.

**Review by envyaims — APPROVED** [review]
image

**Pull request merged**